src/views/run-process-panel/run-process-panel-root.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) The Arvados Authors. All rights reserved.
+//
+// SPDX-License-Identifier: AGPL-3.0
+
+import * as React from 'react';
+import { Stepper, Step, StepLabel, StepContent, Button } from '@material-ui/core';
+
+export interface RunProcessPanelRootDataProps {
+    currentStep: number;
+}
+
+export interface RunProcessPanelRootActionProps {
+    onClick: (step: number) => void;
```

Comment:
maybe  it would be better to name it `onSetStep` ?

Reply:
It would be ;) I'm changing...